addons/sourcemod/data/vip/cfg/info.ini

```diff
@@ -2,24 +2,26 @@
 {
 	"connect_info_time"	// Сообщение для VIP-игрока при входе на сервер (когда вип на время)
 	{
-		//	{NAME}	- Ник игрока
-		//	{GROUP}	- Группа игрока
+		//	{NAME}		- Ник игрока
```

Comment:
comments in this file also need to be done in two languages

Reply:
Done